src/containers/WorkspaceRecordingsView.tsx

```diff
@@ -0,0 +1,51 @@
+import { Button } from '@material-ui/core';
```

Comment:
This is like the home page for a workspace, displaying the table of recordings. It uses workspaceRouteDispatch for navigation.